s3proxy/internal/router/router.go

```diff
@@ -41,12 +55,24 @@ var (
 // Router implements the interception logic for the s3proxy.
 type Router struct {
 	region string
+	kek    []byte
```

Comment:
Could we maybe simply use `[32]byte` as a type? 
The downside is future crypto/key agility.

Reply:
If we want to change from that keysize we are probably also changing the crypto primitive. So a software update will be necessary anyways.

Changed.